.pre-commit-config.yaml

```diff
@@ -0,0 +1,45 @@
+repos:
+  - repo: meta
+    hooks:
+      - id: check-hooks-apply
+      - id: check-useless-excludes
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.5.0
+    hooks:
+      - id: check-yaml
+      - id: end-of-file-fixer
+      - id: trailing-whitespace
+  - repo: https://github.com/golangci/golangci-lint
+    rev: v1.55.2
+    hooks:
+      - id: golangci-lint
+  - repo: https://github.com/igorshubovych/markdownlint-cli
+    rev: v0.37.0
+    hooks:
+      - id: markdownlint
+  - repo: https://github.com/mrtazz/checkmake
+    rev: 0.2.2
+    hooks:
+      - id: checkmake
+        exclude: '^Dockerfile.make$' # actually a Dockerfile and not a makefile
+  - repo: https://github.com/hadolint/hadolint
+    rev: v2.12.1-beta
+    hooks:
+      - id: hadolint
+  - repo: https://github.com/ansible-community/ansible-lint
+    rev: v6.21.1
+    hooks:
+      - id: ansible-lint
```

Comment:
There is no ansible code in the repo AFAIK.

Reply:
Ah I thought of infrastructure while adding them. Good catch 👍